File: Tabel Aritmetika/Tabel-Aritmetika.py
def tabel_penjumlahan():
    batas = 0
    bilangan = 0

    print ("Masukkan bilangan: ", end="")
    bilangan = float (input ())
    print ("Masukkan batas hitung: ", end="")
    batas = int (input ())

    for i in range(1, batas + 1):
        print(f"{bilangan:.2f} + {i} = {bilangan + i:.2f}")

def tabel_pengurangan():
    batas = 0
    bilangan = 0

    print("Masukkan bilangan: ", end="")
    bilangan = float(input())
    print("Masukkan batas hitung: ", end="")
    batas = int(input())

    for i in range(1, batas + 1):
        print(f"{bilangan:.2f} - {i} = {bilangan - i:.2f}")

def tabel_perkalian():
    batas = 0
    bilangan = 0

    print("Masukkan bilangan: ", end="")
    bilangan = float(input())
    print("Masukkan batas hitung: ", end="")
    batas = int(input())

    for i in range(1, batas + 1):
        print(f"{bilangan:.2f} x {i} = {bilangan * i:.2f}")

def tabel_pembagian():
    batas = 0
    bilangan = 0

    print("Masukkan bilangan: ", end="")
    bilangan = float(input())
    print("Masukkan batas hitung: ", end="")
    batas = int(input())

    for i in range(1, batas + 1):
        print(f"{bilangan:.2f} : {i} = {bilangan / i:.2f}")

def main():
    pilihan = 0

    while True:
        print("========== Pembuat Tabel Aritmetika ==========")
        print("1. Penjumlahan")
        print("2. Pengurangan")
        print("3. Perkalian")
        print("4. Pembagian")
        print("5. Keluar")
        pilihan = int(input("Pilih salah satu: "))

        if pilihan == 1:
            tabel_penjumlahan()
        elif pilihan == 2:
            tabel_pengurangan()
        elif pilihan == 3:
            tabel_perkalian()
        elif pilihan == 4:
            tabel_pembagian()
        elif pilihan == 5:
            print("Terima kasih telah menggunakan program ini")
            break
        else:
            print("Input error")

if __name__ == "__main__":
    main()

File: Tabel Aritmetika/Tabel-Aritmetika.c
#include <stdio.h>

void tabel_penjumlahan ();
void tabel_pengurangan ();
void tabel_perkalian ();
void tabel_pembagian ();

int main()
{
    unsigned short pilihan = 0;

    do
    {
        printf ("========== Pembuat Tabel Aritmetika ==========\n");
        printf ("1. Penjumlahan\n");
        printf ("2. Pengurangan\n");
        printf ("3. Perkalian\n");
        printf ("4. Pembagian\n");
        printf ("5. Keluar\n");
        printf ("Pilih salah satu: ");
        scanf ("%hu", &pilihan);

        switch (pilihan)
        {
            case 1:
                tabel_penjumlahan ();
                break;
            case 2:
                tabel_pengurangan ();
                break;
            case 3:
                tabel_perkalian ();
                break;
            case 4:
                tabel_pembagian ();
                break;
            case 5:
                printf ("Terima kasih telah menggunakan program ini\n");
                break;
            default:
                printf ("Input error\n");
                break;
        }
    } while (pilihan > 5 || pilihan < 0);

    return 0;
}

void tabel_penjumlahan ()
{
    float batas = 0, bilangan = 0;

    printf ("Masukkan bilangan: ");
    scanf ("%f", &bilangan);
    printf ("Masukkan batas hitung: ");
    scanf ("%f", &batas);

    for (int i = 1; i <= batas; i++)
    {
        printf ("%.2f + %d = %.2f\n", bilangan, i, bilangan + i);
    }
}

void tabel_pengurangan ()
{
    float batas = 0, bilangan = 0;

    printf ("Masukkan bilangan: ");
    scanf ("%f", &bilangan);
    printf ("Masukkan batas hitung: ");
    scanf ("%f", &batas);

    for (int i = 1; i <= batas; i++)
    {
        printf ("%.2f - %d = %.2f\n", bilangan, i, bilangan - i);
    }
}

void tabel_perkalian ()
{
    float batas = 0, bilangan = 0;

    printf ("Masukkan bilangan: ");
    scanf ("%f", &bilangan);
    printf ("Masukkan batas hitung: ");
    scanf ("%f", &batas);

    for (int i = 1; i <= batas; i++)
    {
        printf ("%.2f x %d = %.2f\n", bilangan, i, bilangan * i);
    }
}

void tabel_pembagian ()
{
    float batas = 0, bilangan = 0;

    printf ("Masukkan bilangan: ");
    scanf ("%f", &bilangan);
    printf ("Masukkan batas hitung: ");
    scanf ("%f", &batas);

    for (int i = 1; i <= batas; i++)
    {
        printf ("%.2f : %d = %.2f\n", bilangan, i, bilangan / i);
    }
}
